src/main/kotlin/lotto/WinningStatistics.kt

```diff
@@ -1,6 +1,16 @@
 package lotto
 
-class WinningStatistics(private val rankCount: Map<Rank, Int>) {
+class WinningStatistics(lottos: List<Lotto>, winningLotto: WinningLotto) {
+    private val rankCount = Rank.entries.associateWith { DEFAULT }.toMutableMap()
+
+    init {
+        lottos.forEach { lotto ->
+            val matchCount = winningLotto.matchCount(lotto)
+            val rank = Rank.valueOf(matchCount, winningLotto.containBonusNumber(lotto))
+            rankCount[rank] = rankCount[rank]?.plus(PLUS) ?: PLUS
+        }
```

Comment:
I think your code was really well structured and easy to understand and build on 👍